catenax/.htaccess

```diff
@@ -87,5 +87,9 @@ RewriteRule ^next/taxonomy(/|\.ttl)?$ https://raw.githubusercontent.com/catenax-
 # Rewrite rule to serve complete taxonomy
 RewriteRule ^taxonomy(/|\.ttl)?$ https://raw.githubusercontent.com/catenax-ng/product-ontology/v23.12/taxonomy.ttl [R=302,L]
 
+# ODRL Profile definition
+RewriteRule ^policy(/|.*)$ https://raw.githubusercontent.com/catenax-eV/cx-odrl-profile/main/profile.ttl [R=302,L]
```

Comment:
Note that when written like this the `/` part is not needed since `.*` will catch everything.  Which also means `policyfoobar` will match.  The expected forms will match too so likely no real harm.  Fun with regexes.

Reply:
ohh, right. good point! It's not relevant right now, but with future changes I might make this clear. Thanks for the hint!